hicommonwealth/commonwealth
# Revert avatar updates

Reverts the updated sizing of avatars and fixes the bugs outlined in #4750.

## Link to Issue
Closes: #4750 and #3934

## Description of Changes
- fixes a couple bugs involving avatars

## "How We Fixed It"
- reverts the initial avatar updates commit
- adds css to handle thread lines issue

## Test Plan
- check to see that comment thread lines are not visible under or over the avatars positioned above them
- check that, when hovering over the thread author avatar, the avatar in the popover fits neatly in its outlining circle

## Other Considerations
- We'll want new deigns around thread comment spacing as the shift in thread lines leaves space a little tight
